lang/en.yml

```diff
@@ -7,6 +7,6 @@ en:
     COLUMN_TYPE: Type
     SELECT_EMPTY: Select…
     SELECT_TYPE: 'Select a content type'
-    TAB_OTHERS: Others
+    TAB_OTHERS: Other
```

Comment:
Can you create a new key? `TAB_OTHER` seems like a nice name

Reply:
But we're changing an existing label to something else, seems like not updating the key is the correct thing.  Also it'll mean that any other languages will loose their translation.  I think we should keep the slightly weird key as is.